misc_docs/syntax/decorator_uncurry.mdx

```diff
@@ -0,0 +1,36 @@
+---
+id: "uncurry-decorator"
+keywords: ["uncurry", "decorator"]
```

Comment:
before i forget, one of the keywords should probably also be `@bs.uncurry`, due to the fact that we will have a lot of users on old versions of ReScript / older blog posts that still use the `@bs.` prefix convention.

Not a blocker, just wanted to mention